Lombiq.HelpfulLibraries/Libraries/Database/SchemaBuilderExtensions.cs

```diff
@@ -51,5 +53,34 @@ public static ISchemaBuilder AddDatabaseIndex<TTable>(this ISchemaBuilder schema
                     $"IDX_{typeof(TTable).Name}_{string.Join("_", columnNames)}",
                     columnNames));
         }
+
+        public static ISchemaBuilder CreateMapIndexTable<T>(
+            this ISchemaBuilder builder,
+            string collection = null) =>
+            builder.CreateMapIndexTable(
+                typeof(T),
+                table =>
+                {

```

Comment:
I think you could format this the same as in line 67:
```suggestion
            builder.CreateMapIndexTable(typeof(T), table =>
            {

```

Reply:
No, because this has an additional parameter after the lambda as well.

Reply:
😢